### ISSUE TITLE
Large grenade keeps flashing

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/yogstation#2380]
Somebody threw an activated large grenade (5 sec timer, maybe it contained nothing), and it kept flashing and not going away for minutes...

**Comment by Time-Green** [comment]
Yeah so, this happens when the beakers inside get removed. It may not have a system in place to check that...
